client/me/security-u2f-key/index.jsx

```diff
@@ -7,6 +7,7 @@ import Gridicon from 'gridicons';
 import React, { Fragment } from 'react';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
+import { isSupported } from 'u2f-api';
```

Comment:
Where is this method coming from?  Or will it be added to the u2f-api in the future?